P2-Proposals/Team - Vanquish/Project2.md

```diff
@@ -0,0 +1,60 @@
+# Poke-Pictionary
+## Vanquish
+    - Osman Elmahadi
+    - Sean Letts
+    - Randy Robinson
+    - Steve Burgos
+## Project Synopsis
+
+A user will be able to log in to our website. From there, they can either look at their past drawings, look at other people's drawings, organized by
+either time of posting (by most recent) or by most popular, able to rate each other's drawings. If they desire, the user can also play a game.
+In single player, the user will recieve a random pokemon. From there, they attempt to draw said pokemon in the canvas screen. When done, their drawing
+is put side by side with the original art. The user can then download it, upload it and save it, or delete it. The user can attempt this with a hint,
+the pokemon sprite next to their drawing, or blind. A 1 on 1 feature could be added where a person attempts to draw a pokemon within a set period of time.
+Once done, the other person gets the image and can try and guess what pokemon it is that the other person drew up. 
+
+## Flowchart 
```

Comment:
@sletts the proposal LGTM, but you might want to remove the Flowchart section as you don't have it.